Resources/config/cache_manager.xml

```diff
@@ -1,15 +1,14 @@
 <?xml version="1.0" ?>
 
 <container xmlns="http://symfony.com/schema/dic/services"
-    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
```

Comment:
not sure about indenting this more. i think 4 spaces is good. not sure what xml conventions in symfony are in general - but the other service files also have just 4 spaces. lets check that separately and if we need to change, we should change consistently.